### DISCUSSION TITLE
小米手環怎麼分析睡眠

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS4719-C/Python-Community-GS4719-C#1003] [General]
雖然我沒有小米手環，但還是很好奇小米手環到底是怎麼分析使用者的睡眠時間跟品質的
小小一個手環竟然可以做那麼多的事，十分有趣。
[https://www.doyouknow.wiki/a/202107/618171.html](url)

**Comment by peggy0329** [comment]
我比較好奇的是小米手環功能蠻實用的，但他售價為何可以如此親民
p.s.網址好像失效了

**Comment by swerfg03joe** [comment]
網址點進去只有404，我是真的好奇分析睡眠時間和品質有甚麼用途，是收集到的數據還能拿去作睡眠醫療的分析嗎?如果只是單純的分析紀錄那好像沒有甚麼太大的價值

**Comment by Sylvie0601** [comment]
我是超級好奇小米手環怎麼分辨的，但我覺得小米手環監測睡眠品質跟甚麼時候入眠可以讓我更清楚昨天睡眠品質如何、睡覺時長等等的功能，可以讓我更能監管自己的健康狀態，或是透過長期的觀察了解自己可以在甚麼時候入睡可以達到最好的品質等等的。

**Comment by manipulateed** [comment]
其實就像醫院的設備可以監測呼吸心律與睡眠一樣的概念，只是透過科技縮小到一隻手環就可以實現的地步，所以未來更有可能裝個芯片就能掌管全身狀況了呢。

**Comment by wenchi9023** [comment]
我自認自己是健康寶寶，睡眠至少都會有7小時
但用了小米手環後發現我的深層睡眠時間只有大概90分鐘，不到總長度的25% (一般人深層睡眠占總睡眠比例)
不知道到底準不準QQ
*網址好像失效了!

**Comment by Guagua1234** [comment]
身為一個有小米手環的同學，我認為小米手環分辨睡眠品質是有一定程度的準確性，在壓力偏大的時候睡眠評分也會備註壓力過大導致深眠時間較短，我也十分好奇他的運作方法，而小米手環也讓我更了解自己的身體狀況，像是睡眠時長多久對我來說才夠，在何時入眠才是最適合我的，長時間監測也可以了解自己的身體狀況，可以從睡眠分數來得知自己最近是不是有潛在的壓力導致自己睡眠品質不佳。

**Comment by BlahFish1006** [comment]
昨天上課老師好像有提到，智慧手環會利用光感測來判斷一個人的呼吸、心跳等生理數據，來推算不同的睡眠周期和睡眠品質。
原理簡單來說是光會從手腕一端照射，中間會通過血管系統，從手腕另一端接收剩餘的光，從而算出血管和身體組織的吸光量，
用一些換算公式去掉其他干擾因素後 (如扣掉血管以外的身體組織)，就能知道血管中帶氧紅血球的吸光量，算出身體的血氧濃度，
再通過血氧濃度回推身體的活動狀態。
( 可惜昨天上課再偷複習考試沒有認真聽qq 本生不代表老師立場XD 希望沒有講錯太多 )

**Comment by orianna901001** [comment]
當初買小米手環，是為了監測自己的睡眠狀況，但其實我個人除了很喜歡她的偵測功能，也覺得她尋找手機的功能很讚，因為我是只要一出門或一回到家，手機常常會亂丟，然後找半個小時不止。

**Comment by BOSENCHANG2004** [comment]
聽說小米手錶會去偵測睡眠過程中的身體活動與姿勢變化（例如翻身動作）。以及持續測量睡眠中的心跳速率與變化，用來判斷睡眠深淺階段。還有追蹤睡眠中的血氧濃度，進一步分析睡眠呼吸情形（如小米手環7、8 pro等型號）。來分析使用者的睡眠狀態
